plugin-animation/src/test/java/com/mapbox/maps/plugin/animation/CameraAnimationsListenersTest.kt

```diff
@@ -172,24 +172,27 @@ class CameraAnimationsListenersTest {
     ) {
       duration = 0
     }
-    val valuesList = mutableListOf<Int>()
+    val valuesList = mutableListOf<String>()
     val listener1 = ValueAnimator.AnimatorUpdateListener {
-      valuesList.add(1)
+      valuesList.add("listener1")
     }
     val listener2 = ValueAnimator.AnimatorUpdateListener {
-      valuesList.add(2)
+      valuesList.add("listener2")
     }
     bearingAnimator.addUpdateListener(listener1)
     bearingAnimator.addUpdateListener(listener2)
-    cameraAnimationsPluginImpl.addCameraBearingChangeListener { valuesList.add(0) }
+    cameraAnimationsPluginImpl.addCameraBearingChangeListener { valuesList.add("camera") }
     cameraAnimationsPluginImpl.registerAnimators(bearingAnimator)
 
     Shadows.shadowOf(Looper.getMainLooper()).pause()
     bearingAnimator.start()
     Shadows.shadowOf(Looper.getMainLooper()).idle()
 
     Assert.assertEquals(5, valuesList.size)
-    Assert.assertArrayEquals(intArrayOf(0, 1, 2), valuesList.slice(0..2).toIntArray())
+    Assert.assertArrayEquals(
+      arrayOf("listener1", "listener2", "camera"),
```

Comment:
I hope it is fine to change the order.

Reply:
As discussed in person it's not OK and it's fixed now.